static/app/views/dashboardsV2/widgetCard/dashboardsMEPContext.tsx

```diff
@@ -0,0 +1,37 @@
+import {ReactNode, useState} from 'react';
+
+import {createDefinedContext} from 'sentry/utils/performance/contexts/utils';
+
+interface DashboardsMEPContextInterface {
+  setIsMetricsData: (value?: boolean) => void;
+  isMetricsData?: boolean;
+}
+
+const [_DashboardsMEPProvider, useDashboardsMEPContext, DashboardsMEPContext] =
+  createDefinedContext<DashboardsMEPContextInterface>({
+    name: 'DashboardsMEPContext',
+  });
+
+const DashboardsMEPConsumer = DashboardsMEPContext.Consumer;
+
+function DashboardsMEPProvider({children}: {children: ReactNode}) {
```

Comment:
should we add a description to this context, mentioning what MEP means? it can help other devs to better understand what component does

Reply:
👍 Added in https://github.com/getsentry/sentry/pull/33860